src/components/Review/DecisionArea.tsx

```diff
@@ -1,137 +1,169 @@
-import React, { useEffect, useState } from 'react'
-import { Button, Container, Form, Header, Modal, Radio, Segment, TextArea } from 'semantic-ui-react'
+import React, { useState } from 'react'
+import { Button, Form, Header, Modal, Radio, Segment, TextArea } from 'semantic-ui-react'
 import { useRouter } from '../../utils/hooks/useRouter'
 import { ReviewResponse, ReviewResponseDecision } from '../../utils/generated/graphql'
 import strings from '../../utils/constants'
 import messages from '../../utils/messages'
 import SummaryViewWrapper from '../../formElementPlugins/SummaryViewWrapper'
 import { SummaryViewWrapperProps } from '../../formElementPlugins/types'
 import useUpdateReviewResponse from '../../utils/hooks/useUpdateReviewResponse'
+import { getOperationAST } from 'graphql'
 
-interface DecisionAreaProps {
+type DecisionAreaProps = SummaryViewWrapperProps & {
   reviewResponse: ReviewResponse
-  toggle: boolean
-  summaryViewProps: SummaryViewWrapperProps
+  isConsolidation: boolean
 }
 
 const DecisionArea: React.FC<DecisionAreaProps> = ({
-  toggle,
   reviewResponse,
-  summaryViewProps,
+  isConsolidation,
+  ...summaryViewProps
 }) => {
-  const { updateQuery } = useRouter()
-  const [isOpen, setIsOpen] = useState(false)
-  const [previousToggle, setPreviousToggle] = useState(false)
-  const [review, setReview] = useState(reviewResponse)
-  const updateResponse = useUpdateReviewResponse(reviewResponse.id)
-
-  useEffect(() => {
-    if (toggle != previousToggle) {
-      setReview(reviewResponse)
-      setPreviousToggle(toggle)
-      setIsOpen(true)
-    }
-  }, [toggle])
+  const {
+    query: { openResponse },
+    updateQuery,
+  } = useRouter()
+  const [reviewUpdate, setReviewUpdate] = useState(reviewResponse)
```

Comment:
i think it would be more clear to name this one `reviewResponse` and the prop can be `reviewResponse: initialReviewResponse`

Reply:
Yes, good idea

Reply:
Done: 04741bb